_posts/2022-11-02-relicensing.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: Relicensing MDAnalysis
+---
+
+<img src="https://www.gnu.org/graphics/lgplv3-with-text-154x68.png"
+title="LGPLv3" alt="LGPLv3 logo" style="float: right"/>
+
+We want to **change the license of MDAnalysis** from the [GNU General
+Public License][GPLv2] (GPL v2+) to the more permissive [Lesser GNU
+Lesser Public License][LGPL] (LGPL v3+) license. Both are *open source
+licenses* but the [LGPL][] gives developers more freedom in how they
+license any of their own code that makes use of MDAnalysis.
+
+The license is the legal basis that determines how code can be copied,
+changed, and incorporated into other code. The license is the only
+reason that one can use someone else's code --- it does not matter if
+the code is publicly available somewhere. If it does not have a
+license you cannot use it legally. Therefore, getting the license
+right is very important.
+
+**In order to change the MDAnalysis license, we need the agreement of
+all our contributors to change from GPL to LGPL.**
```

Comment:
todo: also detail how this is a community consultation, if the MDA users don't want this then we won't go ahead